x-pack/plugins/session_view/public/components/detail_panel_alert_group_item/index.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useMemo } from 'react';
+import { EuiIcon, EuiText, EuiAccordion, EuiNotificationBadge } from '@elastic/eui';
+import { Process, ProcessEvent } from '../../../common/types/process_tree';
+import { useStyles } from '../detail_panel_alert_list_item/styles';
+import { DetailPanelAlertListItem } from '../detail_panel_alert_list_item';
+import { ALERT_COUNT_THRESHOLD } from '../../../common/constants';
+import {
+  ALERT_GROUP_ITEM_TEST_ID,
+  ALERT_GROUP_ITEM_TITLE,
+  ALERT_GROUP_ITEM_COUNT,
+} from '../detail_panel_alert_tab/index.test';
+
+interface DetailPanelAlertsGroupItemDeps {
+  alerts: ProcessEvent[];
+  onProcessSelected: (process: Process) => void;
+  onShowAlertDetails: (alertId: string) => void;
+}
+
+/**
+ * Detail panel description list item.
+ */
+export const DetailPanelAlertGroupItem = ({
+  alerts,
+  onProcessSelected,
+  onShowAlertDetails,
+}: DetailPanelAlertsGroupItemDeps) => {
+  const styles = useStyles({});
```

Comment:
declaring like this works?

```js
const styles = useStyles();
```

so it can avoid creating an empty object in the memory on every rerender